### DISCUSSION TITLE
Lazer not launching properly and no log in

### DISCUSSION THREAD

**Discussion by Anymusxd** [ppy/osu#29015] [Q&A] [answered]
Problem details:

There are some problems launching the game, in this case not launching first try, needing to close it from task manager.
Also it just doesn't log in, no error no nothing, the button doesn't do anything
Should add too that it doesn't close 100%, it's still running in the background when exiting

Video or screenshot showing the problem:

[https://imgur.com/a/PvyA2RU](url)

osu! version: Lazer 2024.718.1-lazer

**Comment by Stoppedpuma** [comment, marked as answer]
Can you attach logs using the "Export logs" button found here?
![image](https://github.com/user-attachments/assets/03a5e7ee-8adb-4d50-8d3f-469906046f89)

**Reply by Anymusxd** [reply]
the whole folder?

**Reply by Stoppedpuma** [reply]
It should have created a zip file, uploading that is fine.

**Reply by Anymusxd** [reply]
hold on, wtf, it just resolved itself now

**Reply by Anymusxd** [reply]
ill still share just for future problems ig
[compressed-logs.zip](https://github.com/user-attachments/files/16339831/compressed-logs.zip)
